mozilla-lockbox/lockbox-android
# Opens FAC link only when "256-bit encryption" link is clicked on "You're all set!" page

Fixes #455 

## Issue
Opens FAQ link when clicking anywhere on "Sync between devices with secure 256-bit encryption" icon/text. The problem is the link was styled as Spannable but the click is called for the whole text.

## Solution
Adds a 'ClickableSpan' to call 'onClick' method only when the link (256-bit encryption) is clicked.

## How to test

1. Install and launch Lockbox.
2. Tap on "Get Started" and enter your email.
3. Login with valid credentials.
4. Wait until the "You're all set!" page is displayed.
5. Check if FAQ is opened only when the lick "256-bit encryption" is clicked.

## Screenshots or Videos

[Video](https://drive.google.com/file/d/19FZyr3u7XmRwKWcUFqicUqwggvGhEh6U/view?usp=sharing)

<img src="https://user-images.githubusercontent.com/4621339/54096974-f536c300-438c-11e9-9222-994df51a0bc7.jpg" width="240"/>


## To Do

- add “WIP” to the PR title if pushing up but not complete nor ready for review
  - optional: consider adding instrumentation (integration/UI) tests
- consider running this branch in a debug simulator and check for memory leak notifications or any warnings
